.gitignore

```diff
@@ -18,11 +18,15 @@
 .env.test.local
 .env.production.local
 /.vs
+.idea
 
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+# Only keep 1 lock-file, try not to mix yarn and npm
```

Comment:
Hvorfor yarn + npm?

Reply:
Jeg brukte yarn lokalt, men innså at det er et problem når github action som er satt opp bruker npm. Github action trenger en package-lock.json, mens yarn lager en yarn.lock, så best å bruke npm. La til i gitignore stedet for å slette yarn.lock lokalt